### DISCUSSION TITLE
"Shrunken brains" registration issue seen on EPI spatial normalisation

### DISCUSSION THREAD

**Discussion by IanSOlivier** [HALFpipe/HALFpipe#410] [General] [answered]
Not sure to what degree this is a question more than an issue. There seems to be registration issue with all of the scans I'm running on multiple occasions (all from the same study but run separately for different scanners), for both resting-state and task-based analyses. This has also been noticed by another researcher using some of the same participants but running HALFpipe separately. 

On quality assessment, the brains seem to have "shrunk" where the outline on the EPI SN leaves quite a significant gap between the template (red outline) and image. It was suggested that I enable "recon_all" in the spec.json file, but unfortunately this did not resolve the issue. 

I'm running HALFpipe on a HPC using singularity. There were no notable error messages or complaints that I could see.

Thought I'd post this here in case someone might have had a similar issue or has any suggestions :)
![EPI_SN_registration_issue_HALFpipe](https://user-images.githubusercontent.com/92156038/186372803-f5a74716-9ed8-4b01-b8c9-7c07e11db63b.png)

**Comment by HippocampusGirl** [comment, marked as answer]
Hi @IanSOlivier,

I  cannot find any registration issues with these images.

What is important is not the dark red outline of the skull stripping, but the pastel-colored outlines of the template brain regions and the cortical ribbon. If these match, and they do as far as I can tell, then the registration succeeded.

I would just compensate for the faulty skull stripping of the functional image by applying the standard space brain mask to the final statistical map. That way you can make sure that the additional voxels don't have any effect on your statistics and hypothesis tests.

**Reply by IanSOlivier** [reply]
Hi @HippocampusGirl,

Thank you so much for your help! I will apply the standard brain mask then to the NIfTI z_statmap output. Would that also be acceptable for the ENIGMA projects (this particular data set is part of the task-based fMRI analyses for the ENIGMA-OCD working group.

Thanks again for the clarification! :-)

**Reply by HippocampusGirl** [reply]
Hi @IanSOlivier,

No matter at which stage you apply the mask, the result will be the same.

If you are doing a voxel-wise analysis, I think the cleanest way would be to simply apply the mask to the image-based meta-analysis/mega-analysis results. That way the data is kept as untouched as possible until you are actually testing a hypothesis :-)

**Reply by IanSOlivier** [reply]
Hello @HippocampusGirl 

Okay, great. Unfortunately I'm not doing the actual meta-/mega-analyses, just doing the HALFpipe preprocessing and submitting the output from our specific sites. That means the mask will probably have to be applied at a later stage by the person doing the final analyses and I let them know, rather than myself applying the mask before submitting the output? 

Thanks again for all the help :-)

**Reply by HippocampusGirl** [reply]
Exactly, this is something they can deal with easily :-)
